wzdx/raw_to_standard/navjoy_568.py

```diff
@@ -0,0 +1,326 @@
+    parser.add_argument('navjoyFile', help='navjoy file path')
+    parser.add_argument('--outputDir', required=False,
+                        default='./', help='output directory')
+
+    args = parser.parse_args()
+    return args.navjoyFile, args.outputDir
+
+
+# take in individual message, spit out list of altered unique messages to be translated
+# This function iterates over the list of NUMBERED_KEY_NAMES and correlates them to CORRECT_KEY_NAMES. For each
+# set of numbered key names, generate a copy of the original message, check if the numbered keys exist, if they do
+# then copy those values to the leys in CORRECT_KEY_NAMES. After, if directionOfTraffic yields more than one direction,
+# generate a new message for each direction and save them with the key 'direction'
+
+# TODO: consider deleting all numbered keys after they are copied
+# TODO: consider removing duplicate messages
+def expand_speed_zone(message):
+    try:
+        messages = []
+        for key_set in NUMBERED_KEY_NAMES:
+            if not message.get('data', {}).get(key_set['street_name']):
+                continue
+            new_message = copy.deepcopy(message)
+            for key, value in key_set.items():
+                new_message.get('data', {})[CORRECT_KEY_NAMES[key]] = message.get(
+                    'data', {}).get(value)
+
+            directions = new_message.get('data', {}).get('directionOfTraffic')
+            for direction in get_directions_from_string(directions):
+                new_message_dir = copy.deepcopy(new_message)
+                if new_message_dir.get('data', {}).get('directionOfTraffic'):
+                    del new_message_dir.get('data', {})['directionOfTraffic']
+                new_message_dir.get('data', {})['direction'] = direction
+                messages.append(new_message_dir)
+        return messages
+    except Exception as e:
+        logging.error(e)
+        return [message]
+
+
+def get_directions_from_string(directions_string) -> list:
+    if not directions_string or type(directions_string) != str:
+        return []
+
+    directions = []
+
+    # iterate over directions and convert short direction names to WZDx enum directions
+    directions_string = directions_string.strip()
+    for dir in directions_string.split('/'):
+        direction = STRING_DIRECTION_MAP.get(dir.lower())
+        if direction:
+            directions.append(direction)
+
+    return directions
+
+
+def generate_standard_messages_from_string(input_file_contents):
+    raw_messages = generate_raw_messages(input_file_contents)
+    standard_messages = []
+    for message in raw_messages:
+        standard_messages.append(
+            generate_rtdh_standard_message_from_raw_single(message))
+    return standard_messages
+
+
+def generate_raw_messages(message_string, invalid_messages_callback=None):
+    msg_lst = json.loads(message_string)
+    messages = []
+
+    # Loop through all elements and print each element to PubSub
```

Comment:
I don't think we're publishing to PubSub here anymore are we?  Can we update this comment?